File: Codigo/Obligatorio/codigo/ArenaGestorFront/src/app/models/Bands/BandUpdateBandDto.ts
import { BandUpdateArtistDto } from "./BandUpdateArtistDto";

export class BandUpdateBandDto {
    musicalProtagonistId: Number = 0;
    name: String = "";
    startDate: Date = new Date();
    genderId: Number = 0;
    artists: Array<BandUpdateArtistDto> = new Array();
}

File: Codigo/Obligatorio/codigo/ArenaGestor/ArenaGestor.APIContracts/Band/BandUpdateBandDto.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;

namespace ArenaGestor.APIContracts.Band
{
    public class BandUpdateBandDto
    {

        [Required]
        public int MusicalProtagonistId { get; set; }

        [MaxLength(50)]
        [Required]
        public string Name { get; set; }

        [Required]
        public DateTime StartDate { get; set; }

        [Required]
        public int GenderId { get; set; }

        [Required]
        public List<BandUpdateArtistDto> Artists { get; set; }
    }
}
